lib/wasi/src/lib.rs

```diff
@@ -258,6 +258,7 @@ impl WasiRuntimeError {
     }
 }
 
+#[allow(clippy::result_large_err)]
```

Comment:
Do we really want to plaster this everywhere instead of, eg, boxing the offending error variant?

Reply:
Yeah, if it's a public error we can't change it anyway.